app/frontend/src/components/JsonBuilder.vue

```diff
@@ -75,6 +75,12 @@ export default {
     }
   },
   mounted() {
+    // add sample data
+    this.items.push(
+      { key: 'firstName', value: 'Jane' },
+      { key: 'lastName', value: 'Smith' }
+    );
+    // let parent component watch for changes
```

Comment:
A reason this needs to be set in mounted() instead of as the initial value up in data()?

Reply:
good call. i've set it as the intial value in data(). thanks